pkg/pb/synthetic_monitoring/checks.proto

```diff
@@ -462,9 +462,10 @@ message MultiHttpSettings {
 
 // MultiHttpEntry represents a single entry in a MultiHttp check.
 message MultiHttpEntry {
-  MultiHttpEntryRequest            request    = 1 [(gogoproto.jsontag) = "request,omitempty"];    // The request parameters.
-  repeated MultiHttpEntryAssertion assertions = 2 [(gogoproto.jsontag) = "checks,omitempty"];     // Zero or more assertions to be made on the response.
-  repeated MultiHttpEntryVariable  variables  = 3 [(gogoproto.jsontag) = "variables,omitempty"];  // Zero or more variables to be used in the request.
+  MultiHttpEntryRequest            request     = 1 [(gogoproto.jsontag) = "request,omitempty"];    // The request parameters.
+  repeated MultiHttpEntryAssertion assertions  = 2 [(gogoproto.jsontag) = "checks,omitempty"];     // Zero or more assertions to be made on the response.
+  repeated MultiHttpEntryVariable  variables   = 3 [(gogoproto.jsontag) = "variables,omitempty"];  // Zero or more variables to be used in the request.
+  bool                             logResponse = 4 [(gogoproto.jsontag) = "logResponse,omitempty"];  // Whether to log the response body in the output.
```

Comment:
Since there might be multiple requests, wouldn't it be better to call the field `LogResponseBodies` or `LogResponses`. I'm slightly in favor of the former, because it specifies what exactly will get added.

Reply:
I was thinking this would be an option on a per request basis instead of all or nothing

Reply:
Could be kind of a pain to specify for every request, though

Reply:
I thought about that.

Why would anyone want to have the body to be logged? Mostly for debugging. In that case, I imagine they want all the responses.

Why would anyone _not_ want a specific response logged? Because there's something sensitive in there that they don't want uploaded to Loki. In that case, why would they want the _other_ responses logged?

If this is for debugging (the test button), the frontend can rework the request so that it includes the option in each response.

Also, the frontend could have a "log all responses" button that causes all the individual options to be set.

Reply:
Ok yeah I'm convinced. I'll rework it to be a global option